website/content/api-docs/auth/azure.mdx

```diff
@@ -55,6 +55,26 @@ identity federation (WIF).
   retried before producing an error.
 - `max_retry_delay` `(string: '60s')` - The maximum delay, in seconds, allowed before retrying an operation.
 - `retry_delay` `(string: '4s')` - The initial amount of delay, in seconds, to use before retrying an operation. Increases exponentially.
+- `rotation_period` `(integer: 0)` – <EnterpriseAlert product="vault" inline />
```

Comment:
should we make these a reusable partial?

Reply:
yeah, at the very least these specifically, since they will generally be defined in code by a helper function.

Reply:
done - i forgot how easy it was to do